docs/general/administration/hardware-selection.md

```diff
@@ -0,0 +1,177 @@
+
+### Graphics Cards (GPUs)
+
+With modern systems, the media engines are usually located on the GPUs. Therefore, the available hardware acceleration options are determined by the choice of GPU.
+
+Encoder Quality: Intel > Nvidia > AMD > Apple
+
+Intel graphics is recommended over Nvidia graphics because of the quality. AMD or Apple Silicon are not recommended since they produce poor quality H.264 and H.265(HEVC) output.
+
+:::note A Note on AMD AV1 Encoders
+
+With AV1, AMD has significantly improved the quality of their encoders. However, Jellyfin currently (as of 10.8.10) does NOT support transcoding to AV1. Even after Jellyfin adds support for transcoding to AV1, transcoding to H.264 or H.265(HEVC) is still more likely due to AV1 being in the early stages of adoption.
+
+:::
+
+A list of common codecs can be found [here](/docs/general/clients/codec-support/)
+
+The following is a list of codecs Jellyfin supports transocding to:
+
+- H.264 (Most common transcode target)
+- H.265 (Limited supported by clients)
+
+:::note AV1 Support
+
+Jellyfin currently (as of 10.8.10) does NOT support transcoding to AV1 as a target. Source content in AV1 will transcode to other codecs normally if needed. However, this might change in the future.
+
+:::
+
+For decoding support, the more codecs the better. However, there are only a few codecs that media is commonly available in, as listed above.
+
+#### Intel Graphics
+
+Please refer to [this table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.
+
+:::caution F series CPUs
+
+Intel CPUs with a model that ends in F don't have integrated graphics. These are intended to be used with a dedicated graphics card.
+
+:::
+
+:::note Intel 12th Gen, 13th Gen and ARC Support
+
+If you are planning to use Linux with Intel 12/13th Gen integrated graphics or ARC, these GPUs only work on Linux Kernel 6.2 or newer. Please check your distribution to make sure it has a supported Linux Kernel version. It is also recommended that `jellyfin-ffmpeg6` be installed instead of the default `jellyfin-ffmpeg5` on these hardware platforms.
+
+:::
+
+#### Nvidia Graphics
+
+Please refer to [this table](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) for supported codecs.
+
+:::caution Low End Cards
+
+Certain low end cards (e.g. GT 1030) don't have encoding hardware. Please be careful when choosing a GPU.
+
+:::
+
+#### AMD Graphics
+
+Please refer to [this article](https://en.wikipedia.org/wiki/Video_Coding_Engine) for supported codecs.
+
+:::caution Low End Cards
+
+Certain low end cards (e.g. RX 6400, RX 6500) don't have encoding hardware. Please be careful when choosing a GPU.
+
+:::
+
+:::caution Integrated Graphics
+
+Many AMD CPUs don't have integrated graphics. These are intended to be used with a dedicated graphics card. Please check the product page for each CPU for more information.
```

Comment:
Might want to add a note which CPU series include a GPU (e. g. the `G` denoted and Zen4)